front/src/routes/scene/edit-scene/actions/BlinkLightParams.jsx

```diff
@@ -6,11 +6,15 @@ import Select from 'react-select';
 class BlinkLight extends Component {
   getOptions = async () => {
     try {
-      const devices = await this.props.httpClient.get('/api/v1/device', {
+      const lightDevices = await this.props.httpClient.get('/api/v1/device', {
         device_feature_category: 'light',
         device_feature_type: 'binary'
       });
-      const deviceOptions = devices.map(device => ({
+      const switchDevices = await this.props.httpClient.get('/api/v1/device', {
+        device_feature_category: 'switch',
+        device_feature_type: 'binary'
+      });
+      const deviceOptions = [...lightDevices, ...switchDevices].map(device => ({
```

Comment:
Maybe let's add a sort here so it's sorted by name ?

Reply:
Fixed